### PR TITLE
WRO-659: Removed `EnzymeAdapterPlugin` and related modules

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Remove enzyme support and dependencies in 5.0.0.

### Resolution
* Removed EnzymeAdapterPlugin and related modules

### Links
WRO-659

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
Please update the PR description.

File: LICENSE [hunk 1]
```diff
@@ -200,9 +200,6 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-With portions of code from enzyme:
-https://github.com/airbnb/enzyme
-
 The MIT License (MIT)
```
Comment (MikyungKim) [inline LICENSE:203]: Remove all contents from below.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# unreleased
+
+* Removed `EnzymeAdapterPlugin` and related modules.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
* Removed `EnzymeAdapterPlugin`.
```

File: LICENSE (hunk 1 above)
Comment (hong6316) [reply LICENSE]: OK

File: CHANGELOG.md (hunk 2 above)
Comment (hong6316) [reply CHANGELOG.md]: OK

**Comment by MikyungKim** [comment]
Please update the branch with the recent develop.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**